presentation/common/src/presentation-common/content/ContentTraverser.ts

```diff
@@ -299,29 +299,34 @@ function traverseContentItemPrimitiveFieldValue(visitor: IContentVisitor, fieldH
   visitor.processPrimitiveValue({ field: fieldHierarchy.field, valueType, namePrefix, rawValue, displayValue });
 }
 
-function createFieldHierarchies(fields: Field[]) {
+/**
+ * `ignoreCategories` parameter enables adding all of the `nestedFields` to parent field's `childFields`
+ *  without considering categories.
+ * @alpha
+*/
+export function createFieldHierarchies(fields: Field[], ignoreCategories?: Boolean) {
   const hierarchies = new Array<FieldHierarchy>();
-  const visitField = (category: CategoryDescription, field: Field, parentField: Field | undefined): FieldHierarchy | undefined => {
+  const visitField = (category: CategoryDescription, field: Field, parentField: Field | undefined, ignoreCategoriesFlag?: Boolean): FieldHierarchy | undefined => {
     let childFields: FieldHierarchy[] = [];
     if (field.isNestedContentField()) {
       // visit all nested fields
-      childFields = visitFields(field.nestedFields, field);
+      childFields = visitFields(field.nestedFields, field, ignoreCategories);

```

Comment:
The flag is available in scope, right? I don't think we need to pass it into these nested functions.

Reply:
Updated